doc/elements-block-header-format.md

```diff
@@ -0,0 +1,742 @@
+| Version             | Yes      | 4 bytes  | `int32_t`                       | little endian | Dynafed version is `0xa0000000` (`2684354560`) which is the normal version `0x20000000` bitwise OR'd with the dynafed mask `0x80000000` <sup>1</sup>         |
+| Previous Block Hash | Yes      | 32 bytes | `uint256`                       | little endian | Hashes are serialized in [reverse byte order](https://bitcoin.stackexchange.com/a/116731/134939)                                                             |
+| Merkle Root         | Yes      | 32 bytes | `uint256`                       | little endian |                                                                                                                                                              |
+| Timestamp           | Yes      | 4 bytes  | `uint32_t`                      | little endian | [Unix time](https://en.wikipedia.org/wiki/Unix_time)                                                                                                         |
+| Block Height        | No       | 4 bytes  | `uint32_t`                      | little endian | Included when option `-con_blockheightinheader=1`, which is active for LiquidV1                                                                              |
+| DynaFed Parameters  | Yes      | Variable | `DynafedParams`                 |               |                                                                                                                                                              |
+| Signblock Witness   | Yes      | Variable | `vector<vector<unsigned char>>` |               | The witness is not serialized [for hashes or weight calculation](https://github.com/ElementsProject/elements/blob/elements-22.1/src/primitives/block.h#L259) |
+
+1. Elements [removes the DynaFed mask](https://github.com/ElementsProject/elements/blob/elements-22.1/src/chain.h#L366) during deserialization. The DynaFed version mask is `0x80000000`.
+
+#### DynaFed Parameters (`class DynaFedParams`)
+
+| Field    | Required | Size | Data Type           | Notes                                                                                                                                            |
+| -------- | -------- | ---- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Current  | Yes      |      | `DynaFedParamEntry` | The active DynaFed parameters at this height                                                                                                     |
+| Proposed | Yes      |      | `DynaFedParamEntry` | The proposed DynaFed parameters for the next [epoch](https://github.com/ElementsProject/elements/blob/elements-22.1/src/chainparamsbase.cpp#L60) |
+
+#### DynaFed Parameters Entry (`class DynaFedParamEntry`)
+
```

Comment:
it would be helpful to add an explanation of when the different entry types appear (e.g. beginning of epochs, proposing and non-proposing epochs)

Reply:
Added some explanation in new commit: https://github.com/ElementsProject/elements/pull/1217/commits/1af35c21de79968db44a2587f3dd3c56333b3063#diff-56e0e05da38ca56b8f9ab1b276cec7acb38f26eeaf3215fa88799e639a67227aR86